.travis.yml

```diff
@@ -3,17 +3,18 @@ language: objective-c
 before_install:
   - brew update
   - brew outdated xctool || brew upgrade xctool
+  - gem install cocoapods
+  - pod install
 xcode_workspace: org.onebusaway.iphone.xcworkspace
 xcode_sdk:
-    #test on latest SDK known to Travis CI
     - iphonesimulator
 xcode_scheme:
     - Debug
-    - AppStoreRelease
```

Comment:
I seem to recall adding this because at one point builds worked fine on debug but not release?